app/scripts/models/auth_brokers/fx-sync-channel.js

```diff
@@ -40,7 +41,9 @@ define(function (require, exports, module) {
     commands: null,
 
     defaultCapabilities: _.extend({}, proto.defaultCapabilities, {
-      sendChangePasswordNotice: true
+      sendAfterSignInConfirmationPollNotice: false,
+      sendAfterSignUpConfirmationPollNotice: false,
```

Comment:
A spot of bike-shedding here with my "ubiquitous language" hat on:

I thought we named sign-in *confirmation* to differentiate it from the *verification* step that happens after sign-up. Are we calling them both "confirmation" everywhere now?